integration/init_test.go

```diff
@@ -82,7 +82,7 @@ func TestIntegration(t *testing.T) {
 		Execute(config.DotnetCoreRuntime)
 	Expect(err).NotTo(HaveOccurred())
 
-	suite := spec.New("Integration", spec.Report(report.Terminal{}), spec.Parallel())
+	suite := spec.New("Integration", spec.Report(report.Terminal{}), spec.Sequential())
```

Comment:
It'd be nice to keep running these tests in parallel. Is it possible to scope the `Sequential()` option to just the specs in the default test?